src/PrestaShopBundle/Controller/Admin/Improve/Design/ThemeController.php

```diff
@@ -220,6 +222,9 @@ public function importAction(Request $request)
 
                 $this->getCommandBus()->handle(new ImportThemeCommand($importSource));
 
+                $event = new ThemeManagementEvent('montheme');
```

Comment:
Why the theme name is hard coded?

Reply:
Because it’s a POC to see if it’s doable ;-) some refacto is needed.